CHANGELOG.md

```diff
@@ -7,11 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Implement cost center autocomplete, lazyload in organization autocomplete and maxHeight prop
+
+### Fixed
+
+- Run schedule job only on saturday
+=======
```

Comment:
This looks weird.

These lines are actually part of a previous commit already merged to the `master` branch, it shouldn't be here.

Could you fix this please?